modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -239,18 +399,91 @@ public void testGetObjectEntry() throws Exception {
 			title, MapUtil.getString(objectEntry.getProperties(), "title"));
 	}
 
+	private void _assertEquals(
+			List<ObjectEntry> expectedObjectEntries,
+			List<ObjectEntry> actualObjectEntries)
+		throws Exception {
+
+		Assert.assertEquals(
+			actualObjectEntries.toString(), expectedObjectEntries.size(),
+			actualObjectEntries.size());
+
+		for (int i = 0; i < expectedObjectEntries.size(); i++) {
+			_assertEquals(
+				expectedObjectEntries.get(i), actualObjectEntries.get(i));
+		}
+	}
+
+	private void _assertEquals(
+			ObjectEntry expectedObjectEntry, ObjectEntry actualObjectEntry)
```

Comment:
Remember to sort alphabetically
```suggestion
			ObjectEntry actualObjectEntry, ObjectEntry expectedObjectEntry)
```